### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by GenevieveBuckley**
Demo gif added to readme.
Development installation instructions updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,24 +7,40 @@ Unsupervised clustering and dataset exploration for high content screens.
...
+
+### License
+
+This project uses the 3-clause BSD license. See `LICENSE.txt`.
+
+### Development installation
+```
+git clone https://github.com/microscopium/microscopium.git
+pip install -r requirements.txt
+cd microscopium
```
Comment (jni) [inline README.md]: cd should come before installing the requirements.

**Comment by jni** [comment]
@GenevieveBuckley you should `git checkout master; git pull upstream master; git push origin master` so that your PRs stop having `#87` on them. I think that's the problem anyway.

File: README.md (hunk 1 above)
Comment (GenevieveBuckley) [reply README.md]: Whoops, thanks!

**Comment by GenevieveBuckley** [comment]
Ok, I've just given that a go now

**Pull request merged**